### PR TITLE
Add airtable api

### PR DESCRIPTION
**Pull request by rui-duan**

### CODE REVIEW

File: pages/api/airtable-create.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  error: string;
+};
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  var Airtable = require("airtable");
+  var base = new Airtable({ apiKey: process.env.AIRTABLE_API_KEY }).base(
+    "appZJm9vl67EuISuq"
```
Comment (NiluK) [inline pages/api/airtable-create.ts:15]: I would recommend moving this api key into a .env file

File: pages/api/airtable-create.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+type Data = {
+  id: string;
+  error: string;
+};
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  var Airtable = require("airtable");
```
Comment (NiluK) [inline pages/api/airtable-create.ts]: generally in js we use `const` and `let` nowadays since `var` is broken

File: pages/api/airtable-create.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  var base = new Airtable({ apiKey: process.env.AIRTABLE_API_KEY }).base(
+    "appZJm9vl67EuISuq"
+  );
+  const ids: any = [];
```
Comment (NiluK) [inline pages/api/airtable-create.ts]: ids should be strings no?

File: pages/api/airtable-create.ts (hunk 2 above)
Comment (NiluK) [reply pages/api/airtable-create.ts]: Also can you please move the require to above the function

**Comment by rui-duan** [comment]
Thanks @NiluK . Comments are addressed.

**Pull request merged**